### DISCUSSION TITLE
Frigate web server went down

### DISCUSSION THREAD

**Discussion by Cold-Lemonade** [blakeblackshear/frigate#6523] [Ask A Question]
I am running Frigate 0.12.0 in a docker container. Earlier today, after it detected a person in the field of view of one of the cameras, Frigate's web server became unavailable (i.e., I could not access the website at port 5000). In fact, I couldn't see the snapshots set over mqtt in my Home Assistant notifications. I know Frigate was still running because I received notifications from my Home Assistant app that are based on mqtt messages from Frigate that had to have been sent after the web server went down. I had to close the docker container and restart it in order to get Frigate's web server running again.

Is there a way for me to recover the old log file? I want to look into what happened.

**Comment by NickM-27** [comment]
by default the log files are stored in memory only

**Reply by blakeblackshear** [reply]
To be clear, containers generally don't retain log messages at all. This is by design.

Frigate outputs all it's log output to stdout which means docker handles capturing and storing log data the way it is designed to. You should be able to find the logs for exited frigate containers the same way you would for any other docker container. There is nothing unique to frigate here.
